Firestore/core/src/firebase/firestore/local/leveldb_migrations.cc

```diff
@@ -112,11 +113,8 @@ void ClearQueryCache(leveldb::DB* db) {
   LevelDbTransaction transaction(db, "Drop query cache");
 
   // Reset the target global entry too (to reset the target count).
-  firestore_client_TargetGlobal target_global{};
-
-  nanopb::StringWriter writer;
-  writer.Write(firestore_client_TargetGlobal_fields, &target_global);
-  transaction.Put(LevelDbTargetGlobalKey::Key(), writer.Release());
+  Message<firestore_client_TargetGlobal> target_global;
+  transaction.Put(LevelDbTargetGlobalKey::Key(), MakeStdString(target_global));
```

Comment:
`Put` has an overload that takes `Message<T>` and does the `MakeStdString` for you already.